daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -24,6 +24,7 @@
 
 import logging
 import os
+import time
```

Comment:
Since this change is in the actual translate step, the PR description is a bit misleading: this should improve not only test stability but translate stability when used by end users.

Reply:
updated :)